packages/material-ui-lab/src/PickersDay/PickersDay.tsx

```diff
@@ -357,14 +358,14 @@ const PickersDay = React.forwardRef(function PickersDay<TDate>(
       centerRipple
       data-mui-test="day"
       disabled={disabled}
-      aria-label={utils.format(day, 'fullDate')}
+      aria-label={!props.children ? utils.format(day, 'fullDate') : undefined}
```

Comment:
```suggestion
      aria-label={!children ? utils.format(day, 'fullDate') : undefined}
```

Let's use the same pattern as for the actual children.